Views/Person/_ListPartial.cshtml

```diff
@@ -0,0 +1,60 @@
+﻿@model IEnumerable<Personaltool.Models.Person>
+
+<table class="table table-hover table-searchable">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.FirstName)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.LastName)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.PersonsMemberStatus)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.PersonsCareerLevels)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.PersonsPositions)
```

Comment:
I think these should have an explicit name, like "Status", "Karriere Level", "Posten" (probably in the Person Model)